tests/backup/backup_helper.go

```diff
@@ -2166,3 +2167,14 @@ func FetchNamespacesFromBackup(ctx context.Context, backupName string, orgID str
 	backedUpNamespaces = resp.GetBackup().GetNamespaces()
 	return backedUpNamespaces, err
 }
+
+// RemoveElementByValue remove the first occurence of the element from the array.Pass a pointer to the array and the element by value.
+func RemoveElementByValue(arr interface{}, value interface{}) {
+	v := reflect.ValueOf(arr).Elem()
```

Comment:
Add a check here to see if the pointer is actually pointing to a slice and return error if it is not. Else it may panic.

Reply:
Added check.